docs/source/examples/dispatch_tutorial.ipynb

```diff
@@ -0,0 +1,513 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "total_demand = 185  # [MW], sets the total demand [units of energy]\n",
+    "demand = (np.sin((hours*np.pi/n_hours*2+phase_shift))*-1+np.ones(N)*(base_shift+1))\n",
+    "\n",
+    "np.random.seed(1234) # sets the seed for repeatability\n",
+    "\n",
+    "noise = np.random.random(N)\n",
+    "demand += noise\n",
+    "\n",
+    "demand = demand/demand.sum() * total_demand  # rescale\n",
+    "\n",
+    "with plt.style.context(\"dark_background\"):\n",
+    "    plt.plot(hours, demand, color='cyan')\n",
+    "    plt.ylabel('Demand [MW]')\n",
+    "    plt.grid(alpha=0.2)\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating the Dispatch Model\n",
+    "\n",
+    "\n",
+    "The simplest model in `osier` requires two things\n",
+    "\n",
+    "1. A **net** demand profile (i.e., if you want to model renewable energy, you should provide a production curve for the given time period of interest).\n",
+    "2. A list of `osier.Technology` objects that can meet the specified demand.\n",
+    "\n",
+    "In order to solve the model, simply call `DispatchModel.solve(solver=\"SOLVER_NAME\")`. This function returns `None`, but you can check that it succeeded by printing `DispatchModel.objective`.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "The default values for the model are:\n",
+    "<ol>\n",
+    "    <li>Energy -- Megawatts [MW]</li>\n",
+    "    <li>Time -- 1 hour [hr] </li>\n",
+    "    <li>Cost -- million dollars per unit [M$/unit]</li>\n",
+    "</ol>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.0041415950000000005\n",
+      "0.4485496000852436\n"
+     ]
+    }
+   ],
+   "source": [
+    "import time\n",
+    "start = time.perf_counter()\n",
+    "model = DispatchModel(technology_list=technology_mix,\n",
+    "                      net_demand=demand\n",
+    "                     )\n",
+    "model.solve(solver='glpk')\n",
+    "end = time.perf_counter()\n",
+    "print(model.objective)\n",
+    "print(end-start)"
+   ]
```

Comment:
I think this section needs to go into more detail on what the individual steps are doing, and the significance of the printed values.

Reply:
I believe this is discussed in the previous markdown cells. The timer should not have to be explained.